docs/ExpectAPI.md

```diff
@@ -1121,6 +1121,20 @@ test('registration applies correctly to orange La Croix', () => {
 });
 ```
 
+Note that you can use any asymmetric matchers to validate specific arguments, combining it with literal values:
+
+```
+expect(func).toHaveBeenCalledWith(
+  expect.anything(),
+  expect.any(MyClass1),
+  42,
+  expect.objectContaining({
+    name: expect.stringContaining('John')
+  }),
+  false
+);
+```
+
```

Comment:
Perhaps:

````suggestion
:::tip

You can also use asymmetric matchers instead of literal values:

```js
expect(func).toHaveBeenCalledWith(
  expect.anything(),
  expect.any(MyClass),
  42,
  expect.objectContaining({
    name: expect.stringContaining('John')
  }),
  false
);
```

:::
````